src/javascript/EditPanel/EditPanelContent/FormBuilder/ChildrenSection/ChildrenSection.jsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {withStyles} from '@material-ui/core';
-import {Typography, Toggle} from '@jahia/design-system-kit';
+import {Typography, Toggle, Badge} from '@jahia/design-system-kit';
 import {ChildrenSectionPropTypes} from '~/FormDefinitions/FormData.proptypes';
 import {ManualOrdering} from './ManualOrdering';
 import {useTranslation} from 'react-i18next';
 import {AutomaticOrdering} from './AutomaticOrdering';
 import {Constants} from '~/ContentEditor.constants';
 import {compose} from '~/utils';
 import {connect} from 'formik';
+import {Public} from '@material-ui/icons';
```

Comment:
same for this icon

Reply:
dont know I reuse the same as the one used for the same badge in Field.jsx